# This service moves the robot towards the gas gradient.

---
uint8 success
